src/dbus2mqtt/dbus/dbus_client.py

```diff
@@ -809,12 +827,77 @@ async def _on_mqtt_msg(self, msg: MqttMessage):
 
                                         try:
                                             logger.info(f"on_mqtt_msg: property={property.property}, value={payload_value}, bus_name={bus_name}, path={path}, interface={interface_config.interface}")
+                                            # converted_property_value = convert_connman_property_args(payload_value)
                                             await self.set_dbus_interface_property(interface, property.property, payload_value)
+                                            
+                                            # Send property set response if configured
+                                            await self._send_mqtt_response(
+                                                interface_config, f"set_{property.property}", [payload_value],
```

Comment:
Two concerns, dbus makes a distinction between methods and properties and `set_` is a dbus-fast specific convention. Better to drop the `set_` and make it explicit in `_send_mqtt_response`. See comment in that function